.vib/keycloak/cypress/cypress/integration/keycloak_spec.js

```diff
@@ -0,0 +1,87 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows login/logout', () => {
+  cy.login();
+  cy.get('#input-error').should('not.exist');
+  cy.get('.dropdown-toggle').click({ force: true });
+  cy.contains('Sign Out').click();
+  cy.contains('#kc-login', 'Sign In');
+});
+
+it('allows changing user data', () => {
+  cy.login();
+  cy.get('.dropdown-toggle').click({ force: true });
+  cy.contains('a', 'Manage account').click({ force: true });
+  cy.contains('p', 'Account console loading').should('not.exist'); //Wait until the loading is finished to continue with actions
+  cy.contains('a', 'Personal').click({ force: true });
+  cy.fixture('users').then((user) => {
+    cy.get('#first-name').clear().type(`${user.newUser.firstName}.${random}`);
+    cy.get('#last-name').clear().type(`${user.newUser.lastName}.${random}`);
+    cy.get('#email-address').clear().type(`${user.newUser.email}`);
+    cy.get('#save-btn').click();
+    cy.contains('.pf-c-alert__title', 'has been updated');
```

Comment:
Again, I think the `.pf-c-alert__title` is not really meaningful (there likely is no other message like that in the UI at the moment). If there is, please disregard my comment 😄